app/screens/post_priority_picker/post_priority_picker.tsx

```diff
@@ -0,0 +1,258 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback, useMemo, useState} from 'react';
+import {useIntl} from 'react-intl';
+import {View} from 'react-native';
+import {useSafeAreaInsets} from 'react-native-safe-area-context';
+
+import FormattedText from '@components/formatted_text';
+import {Screens} from '@constants';
+import {PostPriorityColors, PostPriorityType} from '@constants/post';
+import {useTheme} from '@context/theme';
+import {useIsTablet} from '@hooks/device';
+import useNavButtonPressed from '@hooks/navigation_button_pressed';
+import BottomSheet from '@screens/bottom_sheet';
+import {dismissBottomSheet} from '@screens/navigation';
+import {bottomSheetSnapPoint} from '@utils/helpers';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+import {typography} from '@utils/typography';
+
+import Footer from './footer';
+import PickerOption from './picker_option';
+
+import type {BottomSheetFooterProps} from '@gorhom/bottom-sheet';
+
+export const POST_PRIORITY_PICKER_BUTTON = 'close-post-priority-picker';
```

Comment:
this should be moved to where you are opening the bottom sheet and then use `closeButtonId` in the props for this component, then no need to export the constant